src/modules/commander/commander_params.c

```diff
@@ -623,6 +623,20 @@ PARAM_DEFINE_INT32(NAV_RCL_ACT, 2);
  */
 PARAM_DEFINE_INT32(COM_RCL_EXCEPT, 0);
 
+/**
+ * Datalink loss exceptions
+ *
+ * Specify modes in which datalink loss is ignored and the failsafe action not triggered.
+ *
+ * @min 0
+ * @max 31
```

Comment:
Why is that not 7?

Reply:
Should be 7 yeah 🤔 
Probably originated in copy pasting my mistake here: https://github.com/PX4/PX4-Autopilot/pull/17785/files#diff-98b4e0f87126f3486c8dda8f693ee45c682a856f17958aed5a33d51e5fd27766R885

Reply:
@sfuhrer I also wonder why. Who did this change???

Reply:
Do we want to change fix this here or in a separate PR?